bin/foreman-proxy-certs-generate

```diff
@@ -20,20 +20,33 @@ class ForemanProxyCertsGenerate
 
   private
 
+  # rubocop: disable Metrics/MethodLength
   def generate_answers_file
-    temp_file('foreman-proxy-certs-answer',
-              <<-EOF
+    if File.file?('/etc/foreman-installer/scenarios.d/last_scenario.yaml')
```

Comment:
If you want, it would be nice if this was `./config/last_scenario.yaml`, and ditto down below on line 26.

And then open another PR to add foreman-proxy-certs-generate to this sed in the packaging:
   https://github.com/Katello/katello-packaging/blob/master/katello-installer/katello-installer-base.spec#L96

The reason the paths are like this - including the INSTALLER_DIR on line 7, is it *should* be possible to run the installer out of any directory.  It used to be that way but it's broken for a bunch of reasons - one of them is we have hard coded `/etc/foreman-installer` paths like this in some places.

We need to go back and fix it, but for now if we get it right on new stuff, that'd be great.